UI/Base/ScrollableFullScreen.ts

```diff
@@ -8,25 +8,27 @@ import Ornament from "./Ornament";
  * Wraps some contents into a panel that scrolls the content _under_ the title
  */
 export default class ScrollableFullScreen extends UIElement {
-    private _component: Combine;
+    private _component: UIElement;
 
 
     constructor(title: UIElement, content: UIElement) {
         super();
-        const returnToTheMap = Svg.back_svg().onClick(() => {
+        const returnToTheMap = Svg.back_ui().onClick(() => {
             State.state.fullScreenMessage.setData(undefined);
             State.state.selectedElement.setData(undefined);
-        }).SetClass("featureinfobox-back-to-the-map only-on-mobile mb-2")
+        }).SetClass("block sm:hidden mb-2 bg-blue-50 rounded-full w-12 h-12 p-1.5")
```

Comment:
This change breaks the landscape view.

![image](https://user-images.githubusercontent.com/111561/105640625-55e72000-5e7f-11eb-8b54-2b8d3b389aa1.png)

Right now, the way the Landscape view is build, one cannot get out of the dialogue.

How important is the landscape view? Could we maybe ditch it and tell people to hold the phone horizontally? If not, we should research more about custom breakpoints in Tailwind so we can maybe add them to the config so all the helper-classes get generated automatically. I never read about a height-based breakpoint though …